code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1518,3 +1518,18 @@
 	drink_name = "Pina Colada"
 	drink_desc = "After taking a sip, you feel contractually obligated to start singing a certain song of the same name."
 	taste_description = "tart and tropical pineapple"
+
+/datum/reagent/consumable/ethanol/bilkshake
+	name = "Bilkshake"
+	id = "bilkshake"
+	description = "An upsetting treat that combines beer and milk."
+	reagent_state = LIQUID
```

Comment:
```suggestion
```
already handled by `ethanol` parent type. This should be the last thing to change I think